content/hi/devsecops.md

```diff
@@ -0,0 +1,31 @@
+---
+title: DevSecOps 
+status: Completed
+category: concept
+tags: ["methodology", "", ""]
+---
+## यह क्या है
+
+DevSecOps शब्द विकास, संचालन और सुरक्षा जिम्मेदारियों के सांस्कृतिक विलय को संदर्भित करता है।
+यह सुरक्षा प्राथमिकताओं को शामिल करने के लिए [DevOps](/devops/) दृष्टिकोण का विस्तार करता है
+डेवलपर और परिचालन कार्यप्रवाह में न्यूनतम या बिना किसी व्यवधान के।
+DevOps की तरह, DevSecOps एक सांस्कृतिक बदलाव है, जिसे अपनाई गई तकनीकों द्वारा अद्वितीय अपनाने के तरीकों के साथ आगे बढ़ाया गया है।
+
+## समस्या यह संबोधित करता है
+
+DevOps प्रथाओं में [निरंतर एकीकरण] (/ निरंतर-एकीकरण /) और [निरंतर परिनियोजन] (/ निरंतर-वितरण /) शामिल हैं
+और अनुप्रयोग विकास और रिलीज चक्रों में तेजी लाएं।
+दुर्भाग्य से, स्वचालित रिलीज़ प्रक्रियाएं जो प्रतिनिधित्व करने में विफल होती हैं
+सभी संगठनात्मक हितधारक पर्याप्त रूप से मौजूदा मुद्दों को बढ़ा सकते हैं।
+एक प्रक्रिया जो सुरक्षा आवश्यकताओं पर विचार किए बिना नए सॉफ़्टवेयर को तेज़ी से रिलीज़ करती है
+किसी संगठन की सुरक्षा मुद्रा को नीचा दिखा सकता है।
+
+## यह कैसे मदद करता है
+
+DevSecOps टीम साइलो को तोड़ने पर ध्यान केंद्रित करता है और सुरक्षित, स्वचालित वर्कफ़्लोज़ के निर्माण को बढ़ावा देता है।
+सुरक्षा अनुप्रयोगों का चयन करते समय, संगठनों को इसका लाभ उठाना चाहिए
+स्वचालित CI/CD कार्यप्रवाह और नीति प्रवर्तन जो डेवलपर को सशक्त बनाते हैं।
+लक्ष्य अवरोधक बनना नहीं बल्कि सुरक्षा नीतियों को लागू करना है
+उपयोगकर्ताओं को अपने प्रोजेक्ट को आगे बढ़ाने के तरीके के बारे में सटीक जानकारी देते हुए।
+जब ठीक से लागू किया जाता है, तो एक संगठन बेहतर टीम संचार प्राप्त करेगा और
+सुरक्षा दुर्घटनाओं और संबंधित लागतों को कम करें।
```

Comment:
The alignment of sentences can be fixed. The original paragraph has sentences in continuation, please fix the extra new line and spaces.